### PR TITLE
Updates Agent file 

### PR DESCRIPTION
**Pull request by jonathansheets517**
__init__ command need to call def _copy_ai_settings(self): and def _copy_prompt(self):

Updated lines added with # at the end

    def __init__(self, prompt):
        self.auto_gpt_path = Path(__file__).parent / "Auto-GPT"
        self.auto_workspace = self.auto_gpt_path / "auto_gpt_workspace"
        self.auto_workspace.mkdir(exist_ok=True)  # ADDED this line to make sure auto_gpt_workspace folder exists
        self.prompt_file = self.auto_workspace / "prompt.txt"
        self.output_file = self.auto_workspace / "output.txt"
        self.file_logger = self.auto_workspace / "file_logger.txt"
        self.ai_settings_file = Path(__file__).parent / "AutoGPTData" / "ai_settings.yaml"
        self.ai_settings_dest = self.auto_workspace / "ai_settings.yaml"
        self.prompt = prompt
        self._clean_up_workspace()
        self._copy_prompt()  #Added this too need to copy prompt
        self._copy_ai_settings() #ADDED IT TO COPY AI_SETTING.YAML TO SUBMODULE Auto-GPT workspace

### CODE REVIEW

File: auto_gpt_benchmarking/AutoGPTAgent.py [hunk 1]
```diff
@@ -1,91 +1,92 @@
...
-        print('about to do clean up')
-        print(answer)
-        self._clean_up_workspace()
-        print('did clean up')
-        return answer
-
-
-
-
+"""
```
Comment (dschonholtz) [inline auto_gpt_benchmarking/AutoGPTAgent.py:1]: It's showing the whole file changed here. So I can't really review this

File: README.md [hunk 2]
```diff
@@ -58,13 +53,39 @@ You must have a docker container built corresponding to the submodule below or t
 Cd into the AutoGPT submodule and build/tag the dockerfile so the agent can be instantiated.
 `cd auto_gpt_benchmarks/Auto-GPT`
 
-Build the container so we can run it procedurally!
+Build the container so we can run it procedurally! Make sure to have docker running on your computer.
 `docker build -t autogpt .`
 
+After docker is built, navigate to Auto-GPT\Auto-GPT-Benchmarks directory and set the API Key in command line for windows/mac/linux below:
+
+(Windows) set OPENAI_API_KEY=<insert here>
```
Comment (dschonholtz) [inline README.md]: This looks good. Can you just update it and make sure it doesn't conflict with the minor readme changes I just merged in from your partner in crime?

**Review by dschonholtz — CHANGES_REQUESTED** [review]
This looks good. If you can update it so that it doesn't show every line as changed on the agent file I think you are good for an approve

File: README.md (hunk 2 above)
Comment (jonathansheets517) [reply README.md]: yea sure

**Comment by dschonholtz** [comment]
Closing this in favor with the refactor

**Pull request closed without merging**